Android.bp

```diff
@@ -0,0 +1,67 @@
+cc_library_headers {
+    name: "nuttx_libc_headers",
+    generated_headers: [
+        "nuttx_headers_gen",
+    ],
+    export_generated_headers: ["nuttx_headers_gen"],
+    proprietary: true,
+}
+
+cc_defaults {
+    name: "nuttx_libc_defaults",
+    cflags: [
+        "-DCONFIG_DISABLE_MOUNTPOINT",
```

Comment:
Why not use generated config.h?

Reply:
@anchao，When we compile this library under AOSP, it's possible that NuttX has not been compiled yet, hence the absence of config.h. If we deliberately create one, the content inside cannot represent all compilation environments. so, using flags.

Reply:
Thanks, that makes sense.